pallets/automation-time/src/weights.rs

```diff
@@ -62,6 +63,15 @@ impl<T: frame_system::Config> WeightInfo for AutomationWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(3 as Weight))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
+	// Storage: Timestamp Now (r:1 w:0)
```

Comment:
where did this come from?

Reply:
when i didn't have `#[pallet::weight(<T as Config>::WeightInfo::schedule_transfer_task_existing_slot())]`, i got error: 
```
Invalid pallet::call, requires weight attribute i.e. `#[pallet::weight($expr)]`
```

Reply:
ohh okay. just make sure to replace it w/ your own benchmark before you merge